Dockerfile

```diff
@@ -4,22 +4,6 @@
 ARG base_image=concourse/dev
 FROM ${base_image} AS base
 
-# generate keys (with 1024 bits just so they generate faster)
```

Comment:
These keys are now checked in, so they don't need to be generated in dev builds anymore. This makes it easier to run the tests against both 'dev' images and 'latest' Concourse images (e.g. upgrade/downgrade tests).